reference/api/keys.md

```diff
@@ -242,7 +244,7 @@ Get information on the specified key. Attempting to use this endpoint with a non
     "documents.add"
   ],
   "indexes": [
-    "products"
+    "prod*"
```

Comment:
IMHO, we shouldn't overuse the wildcard and try to include it everywhere. This tends to introduce concepts that are a bit unnecessary to illustrating a task.

In this case, it might even give the idea people should preemptively wildcard everything, which doesn't seem like good advice.

```suggestion
    "products"
```